src/plugins/embeddable/public/lib/embeddables/error_embeddable.tsx

```diff
@@ -47,13 +48,18 @@ export class ErrorEmbeddable extends Embeddable<EmbeddableInput, EmbeddableOutpu
   public render(dom: HTMLElement) {
     const title = typeof this.error === 'string' ? this.error : this.error.message;
     this.dom = dom;
+    const markdownRenderer = markdownFactory();
+    const renderedMarkdown = markdownRenderer(title);
     ReactDOM.render(
       // @ts-ignore
       <div className="embPanel__error embPanel__content" data-test-subj="embeddableStackError">
         <EuiText color="subdued" size="xs">
           <EuiIcon type="alert" color="danger" />
           <EuiSpacer size="s" />
-          {title}
+          <div /* eslint-disable-next-line react/no-danger */
```

Comment:
I haven't pulled this PR down yet, but it seems like the `Markdown` component is already doing exactly this: https://github.com/elastic/kibana/blob/master/src/plugins/kibana_react/public/markdown/markdown.tsx

Is there anything blocking us from using it here?

Reply:
I thought it was an overkill, since the message may not even be a markdown. But on second thought, it makes sense.